### DISCUSSION TITLE
Add achievement "YEET"

### DISCUSSION THREAD

**Discussion by lukepighetti** [community/community#18290] [Other Feature Feedback, Questions, & Ideas] [answered]
YEET

**Comment by schneedotdev** [comment]
Hahaha, what would you have to do to unlock it?

**Reply by lukepighetti** [reply]
I'm not sure, I've been thinking about it all day. Some ideas:

1) force push to branch (main????)
2) delete branch
3) merge main to another branch

**Reply by alice-i-cecile** [reply]
Create a PR that is significantly negative net lines of code!

**Reply by schneedotdev** [reply]
I like that one @alice-i-cecile. That's a winner!

**Reply by lukepighetti** [reply]
I'm a fan!

**Reply by lukepighetti** [reply, marked as answer]
What about -500 (or more) +0

**Reply by SMOSTAFAH1** [reply]
> I'm not sure, I've been thinking about it all day. Some ideas:
> 
> 1. force push to branch (main????)
> 2. delete branch
> 3. merge main to another branch

alias g-res='git pull --rebase ; git checkout --orphan x && git add . && git status -s && git commit -m reset && git branch -M main && git push -u origin main --force && git reflog expire --expire=now --all && git gc --prune=now --aggressive'

**Reply by sergej-stk** [reply]
Force push to main = Yeet :D

**Comment by dr4g0n1x** [comment]
YEET

**Reply by mujahed85** [reply]
YEET

**Reply by lukepighetti** [reply]
YEET (delete 500 lines of code with no additions)
